jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -52,11 +53,23 @@ class SpectralExtraction(PluginTemplateMixin, ApertureSubsetSelectMixin,
     uses_active_status = Bool(True).tag(sync=True)
 
     # feature flag for cone support
-    dev_cone_support = Bool(False).tag(sync=True)
+    dev_cone_support = Bool(False).tag(sync=True)  # when enabling: add entries to docstring
+    dev_bg_support = Bool(False).tag(sync=True)  # when enabling: add entries to docstring
+    dev_subpixel_support = Bool(False).tag(sync=True)  # when enabling: add entries to docstring
+
+    active_step = Unicode().tag(sync=True)
+
     wavelength_dependent = Bool(False).tag(sync=True)
     reference_wavelength = FloatHandleEmpty().tag(sync=True)
     slice_wavelength = Float().tag(sync=True)
 
+    bg_items = List([]).tag(sync=True)
+    bg_selected = Any('').tag(sync=True)
+    bg_scale_factor = Float(1).tag(sync=True)
+    bg_wavelength_dependent = Bool(False).tag(sync=True)
+
+    subpixel = Bool(False).tag(sync=True)
```

Comment:
Reminder: let's change references to `subpixel` to something which is not the name of one of the three aperture masking methods ({exact, subpixel, center}). It's clunky but accurate to call these "aperture masking methods."

Reply:
I'll leave this up to #2679 to modify (with no objections if we do in fact go with three methods instead of just two).